src/coreclr/vm/codeman.cpp

```diff
@@ -1911,6 +1934,41 @@ void EEJitManager::SetCpuInfo()
     CPUCompileFlags.Set64BitInstructionSetVariants();
     CPUCompileFlags.EnsureValidInstructionSetSupport();
 
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+    if (isGenuineIntel)
+    {
+        // Some architectures can experience frequency throttling when executing
+        // executing 512-bit width instructions. To account for this we set the
+        // default preferred vector width to 256-bits in some scenarios. Power
+        // users can override this with `DOTNET_PreferredVectorBitWith=512` to
```

Comment:
`DOTNET_PreferredVectorBitWidth=512`

Reply:
Fixed as part of one of the other two PRs I'm extracting out of the original change.